packages/ERTP/src/purse.js

```diff
@@ -3,20 +3,17 @@ import { AmountMath } from './amountMath.js';
 import { makeTransientNotifierKit } from './transientNotifier.js';
 import { makeAmountStore } from './amountStore.js';
 
-// TODO `InterfaceGuard` type parameter
-/** @typedef {import('@endo/patterns').InterfaceGuard} InterfaceGuard */
-/** @typedef {import('@agoric/zone').Zone} Zone */
-
 const { Fail } = assert;
 
+// TODO Type InterfaceGuard better than InterfaceGuard<any>
```

Comment:
Need help. Finally resorted to `any` after everything else I tried failed. I don't understand why this problem arose in this PR but not prior to this PR.

I got rid of all the `@typedef`s in case it could help with this, which it did not. But I left the non-@typedef form in anyway because it is now our recommended practice. Preserves type info better in the IDE.